### ISSUE TITLE
Discrepancy between SwarmRequest.available_observatories() signature and usage

### ISSUE THREAD

**Issue by lmar76** [ESA-VirES/VirES-Python-Client#53]
According to its signature, SwarmRequest.available_observatories() method allows `collection=None`:

```
Signature:
SwarmRequest.available_observatories(
    self,
    collection=None,
    start_time=None,
    end_time=None,
    details=False,
)
Docstring:
Get list of available observatories from server.

Search availability by collection, one of::

    "SW_OPER_AUX_OBSH2_"
    "SW_OPER_AUX_OBSM2_"
    "SW_OPER_AUX_OBSS2_"

Example usage::

    from viresclient import SwarmRequest
    request = SwarmRequest()
    # For a list of observatories available:
    request.available_observatories("SW_OPER_AUX_OBSM2_")
    # For a DataFrame also containing availability start and end times:
    request.available_observatories("SW_OPER_AUX_OBSM2_", details=True)
    # For available observatories during a given time period:
    request.available_observatories(
        "SW_OPER_AUX_OBSM2_", "2013-01-01", "2013-02-01"
    )

Args:
    collection (str): collection name (e.g. `"SW_OPER_AUX_OBSM2_"`)
    custom_model (str): as with set_products
    details (bool): returns DataFrame if True

Returns:
    list or DataFrame: IAGA codes (and start/end times)
File:      /opt/conda/lib/python3.7/site-packages/viresclient/_client_swarm.py
Type:      function
```

The following code:

```python
request.available_observatories(None, '2016-01-01', '2016-01-02')
```

raises a `ValueError` exception:

```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-93-bf2ce3a7a8e7> in <module>
      2 
      3 request = SwarmRequest()
----> 4 request.available_observatories(None, '2016-01-01', '2016-01-02')

/opt/conda/lib/python3.7/site-packages/viresclient/_client_swarm.py in available_observatories(self, collection, start_time, end_time, details)
    816         ]
    817         if collection not in obs_collections:
--> 818             raise ValueError(f"Invalid collection: {collection}")
    819         if start_time and end_time:
    820             start_time = parse_datetime(start_time)

ValueError: Invalid collection: None
```

**Comment by smithara** [comment]
The reason for this `xxx=None` is in order to specify that these are keyword arguments, but without setting a default behaviour. I'm not sure what is the correct way to set that up with a better docstring (I'll check other libs)... but I've already done that in other places (e.g. https://viresclient.readthedocs.io/en/latest/api.html#viresclient.SwarmRequest.get_between)

I see here I forgot to fix the Args section too!
```
Args:
    collection (str): collection name (e.g. `"SW_OPER_AUX_OBSM2_"`)
    custom_model (str): as with set_products
    details (bool): returns DataFrame if True
```
should be
```
Args:
    collection (str): collection name (e.g. `"SW_OPER_AUX_OBSM2_"`)
    start_time (datetime / ISO_8601 string)
    end_time (datetime / ISO_8601 string)
    details (bool): returns DataFrame if True

```

I imagine there are some other documentation errors / areas to improve. I just made the 0.7.0 release today, so this and other fixes can come in a 0.7.1

**Comment by pacesm** [comment]
Please consider there is already the `SwarmRequest.set_collection()` method. Perhaps you could remove the collection parameter completely.
